documentation.md

```diff
@@ -29,6 +29,7 @@ This page attempts to track the various relevant documentation that exists for R
 * [How to ensure Large File Support for tools using the rpm API](devel_doc/large_files.html)
 
 ## RPM Language Bindings
+* [RPM Python examples](python.html) - A showcase of real-life Python RPM examples
 * [RPM Python](https://web.archive.org/web/20050320013335/http://www.ukuug.org/events/linux2004/programme/paper-PNasrat-1/rpm-python-slides/frames.html) slideset / tutorial by Paul Nasrat (2004)
```

Comment:
It would be nice to not continue linking 20-year-old resources as well, if this new material covers the same ground.